code/modules/virus2/effect/effect.dm

```diff
@@ -941,6 +941,25 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/carbon/mob)
 				mob.apply_damage(10)
 
 
+/datum/disease2/effect/thick_blood
+	name = "Hyper-Fibrinogenesis"
+	stage = 3
+	var/skip = FALSE
```

Comment:
It wouldn't compile without that. The above viruses use it as well, for instance the ones that do thick skin use it to check for mobs with skin.